src/main/java/entities/User.java

```diff
@@ -241,53 +201,51 @@ public List<Timer> getTimers() {
      * Adds an invitation to this User's outgoingInvitations
      * @param invitation The new invitation to add
      */
-//    public void addOutgoingInvitation(Invitation invitation) {
-//        this.outgoingInvitations.add(invitation);
-//    }
+    public void addOutgoingInvitation(Invitation invitation) {
+        this.outgoingInvitations.add(invitation);
+    }
 
     /**
      * Removes an invitation from this User's outgoingInvitations
      * @param invitation The new invitation to remove
      */
-//    public void removeOutgoingInvitation(Invitation invitation) {
-//        this.outgoingInvitations.remove(invitation);
-//    }
+    public void removeOutgoingInvitation(Invitation invitation) {
+        this.outgoingInvitations.remove(invitation);
+    }
 
     /**
      * Gets the list of outgoingInvitations associated with this User
      *
      * @return outgoingInvitations A list of outgoingInvitations associated with this User
      */
-    //   public List<Invitation> getOutgoingInvitations() {
-    //       return this.outgoingInvitations;
-    //   }
+    public List<Invitation> getOutgoingInvitations() {
+        return this.outgoingInvitations;
+    }
 
     /**
      * Adds an invitation to this User's incomingInvitations
      * @param invitation The new invitation to add
      */
-//    public void addIncomingInvitation(Invitation invitation) {
-//        this.incomingInvitations.add(invitation);
-//    }
+    public void addIncomingInvitation(Invitation invitation) {
+        this.incomingInvitations.add(invitation);
+    }
 
     /**
      * Removes an invitation from this User's incomingInvitations
      * @param invitation The new invitation to remove
      */
-//    public void removeIncomingInvitation(Invitation invitation) {
-//        this.incomingInvitations.remove(invitation);
-//    }
+    public void removeIncomingInvitation(Invitation invitation) {
+        this.incomingInvitations.remove(invitation);
```

Comment:
since I sent you a new Invitation(), this will not work. Consult my pull request for alternative way